src/QueryVar.php

```diff
@@ -31,6 +31,17 @@ interface QueryVar {
 	 */
 	const AMP = 'amp';
 
+	/**
+	 * Query var which overrides the template mode to be Standard (aka AMP-first aka canonical).
+	 *
+	 * This is only honored during validation requests or if the user has validation capability.
+	 *
+	 * @see \AMP_Validation_Manager::is_validate_request()
+	 * @see \AMP_Validation_Manager::has_cap()
+	 * @var string
+	 */
+	const AMP_FIRST = 'amp-first';
```

Comment:
Note that the hyphenated `amp-first` is used because it's intended to be user-facing. 

The new `amp_store` query parameter is not user-facing, and goes along with `amp_validate`. So that's why I chose an underscore.